### PR TITLE
Adding info about code push to the faq

### PR DESCRIPTION
**Pull request by sfshaza2**
Code push is supported by Shorebird.

Fixes https://github.com/flutter/website/issues/11062

cc @sethladd

### CODE REVIEW

File: src/content/resources/faq.md [hunk 1]
```diff
@@ -357,6 +356,16 @@ developers to access new mobile OS features and capabilities
 immediately. Developers don't have to wait for the Flutter team
 to expose the new mobile OS capability.
 
+### Does Flutter support code push?
+
+Code push, or the ability to push app updates directly to a user's device,
+isn't directly supported by Flutter. However, there is a third
```
Comment (sethladd) [inline src/content/resources/faq.md:362]: potentially we could say something like "However, we're aware of
at least one third-party solution for code push for Flutter called Shorebird. Note that this is not
an official endorsement or recommendation."

File: src/content/resources/faq.md [hunk 2]
```diff
@@ -357,6 +356,16 @@ developers to access new mobile OS features and capabilities
...
+party solution, called [Shorebird][]. For more information,
+check out the [shorebird_code_push][] package on pub.dev.
```
Comment (sethladd) [inline src/content/resources/faq.md:364]: I suspect their homepage is sufficient for folks who want to learn more. All things being equal, the less links we need to keep updated the better so consider removing the second sentence.

**Review by sethladd — APPROVED** [review]
thanks! optimistically LGTM with comments

**Comment by sfshaza2** [comment]
@domesticmouse, @sethladd has already approved this. Do you mind?

**Review by parlough — APPROVED** [review]

**Comment by sfshaza2** [comment]
thanks, @parlough! Nevermind, @domesticmouse.

**Pull request merged**